travis.sh

```diff
@@ -67,6 +67,59 @@ build_java_oracle8() {
   build_java
 }
 
+build_android() {
```

Comment:
I'd prefer not to introduce any unnecessary dependencies. Android builds are taking 3 minutes right now, which isn't too bad. It's cocoapods that I wish I could speed up for the iOS builds.